test/e2e/storage/volume_provisioning.go

```diff
@@ -144,7 +144,7 @@ func testDynamicProvisioning(t storageClassTest, client clientset.Interface, cla
 
 		path := "/mnt/volume1"
 		pod, err := framework.CreateSecPod(client, claim.Namespace, []*v1.PersistentVolumeClaim{claim}, false,
-			"", false, false, framework.SELinuxLabel, nil, framework.PodStartTimeout)
+			"", t.nodeName, false, false, framework.SELinuxLabel, nil, framework.PodStartTimeout)
```

Comment:
which provisioner test actually passes in a non-nil node name?

Reply:
hostpath (and gce pd) csi tests need it https://github.com/kubernetes/kubernetes/blob/bc98f7a5f2086a62af80ad92f7201e7af5e0ef03/test/e2e/storage/csi_volumes.go#L134

Reply:
I can understand hostpath, not sure about gce. anyway that's a separate issue